### PR TITLE
#162170699 Configure Authors Haven to use PostgreSQL as DB Engine

### PR DESCRIPTION
**Pull request by luqmanoop**
#### What does this PR do?
Configures app to use Postgres as DB Engine and create a basis for setting up the database on other collaborator's machine. Also, I removed the unnecessary mongoose configs & model files.

#### Description of Task to be completed?
- clone the project
- (inside the cloned project) rename `.env.sample` to `.env`
- modify the placeholder contents where relevant

#### How should this be manually tested?
- create a database with the same name set in the environment variable

`cd` into the project (_if not already_) and run the following command.

```
npm run migrate
npm start
```
visit http://localhost:3000

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162170699](https://www.pivotaltracker.com/story/show/162170699)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = {
+    "config": path.resolve('./db/config', 'config.js'),
+    "models-path": path.resolve('./db/models'),
+    "seeders-path": path.resolve('./db/seeders'),
+    "migrations-path": path.resolve('./db/migrations')
+}
```
Comment (omokaro-faith) [inline .sequelizerc]: A line break needed here.

File: .gitignore [hunk 2]
```diff
@@ -34,3 +34,5 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+# custom
+.env
```
Comment (omokaro-faith) [inline .gitignore:38]: A line break needed here.

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,5 @@
+DB_NAME=""
+DB_USER=""
+DB_PASSWORD=""
+DB_HOST=localhost
+DB_DIALECT=postgres
```
Comment (omokaro-faith) [inline .env.sample]: A line break needed here.

File: db/migrations/20181126141320-create-user.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('Users');
+  }
+};
```
Comment (omokaro-faith) [inline db/migrations/20181126141320-create-user.js]: A line break needed here.

**Review by omokaro-faith — CHANGES_REQUESTED** [review]

File: db/migrations/20181126141320-create-user.js (hunk 4 above)
Comment (luqmanoop) [reply db/migrations/20181126141320-create-user.js]: fixed now.

**Pull request merged**